src/gameComponents/DefenseTokens/DefenseToken.java

```diff
@@ -1,23 +1,31 @@
 package gameComponents.DefenseTokens;
 
-import engine.GameConstants;
-
 public class DefenseToken {
-    /**
+    /*
     A defense Token is a token that can be assigned to squadrons or ships. A defense token has a variety of states
     The Defense token class allows for the creation of defense tokens and allows for the various state changes
     it may undergo during normal use
      */
 
+    /**
+     *Defense Token Constants
+     */
+    public enum defenseTokenType {BRACE, REDIRECT, EVADE, SCATTER, CONTAIN, SALVO}
```

Comment:
An enum is kinda like a simple object right?  Do you know if it is convention for enums to be ProperCased like classes?

Reply:
Any objection to simplifying the name so we can do something like DefenseToken.type instead of DefenseToken.defenseTokenType ?

There might be a bit of a name space collision, but I like now much shorter that is, especially since we are likely to be accessing them from outside the class regularly.

Reply:
> Any objection to simplifying the name so we can do something like DefenseToken.type instead of DefenseToken.defenseTokenType ?
> 
> There might be a bit of a name space collision, but I like now much shorter that is, especially since we are likely to be accessing them from outside the class regularly.

Sounds doable. I don't think we end up with much namespace collision, as referring to the object's type its object.type, but getting the enum would be DefenseToken.type. Should be straightforward to do the same to status as well

Reply:
> An enum is kinda like a simple object right? Do you know if it is convention for enums to be ProperCased like classes?

According to https://stackoverflow.com/questions/3069743/coding-conventions-naming-enums/3069863 ProperCase instead of camelCase is the convention

Reply:
granted they're 1 word names now...